trame_server/core.py

```diff
@@ -505,15 +505,15 @@ def force_state_push(self, *key_names):
 
     def start(
         self,
-        port: int = None,
+        port: int | None = None,
         thread: bool = False,
-        open_browser: Optional[bool] = None,
+        open_browser: bool | None = None,
         show_connection_info: bool = True,
         disable_logging: bool = False,
-        backend: str = None,
+        backend: str | None = None,
```

Comment:
may want to use literal instead of str. [aiohttp, generic, tornado, jupyter]
from https://github.com/Kitware/wslink/blob/master/python/src/wslink/backends/__init__.py

Reply:
Thank you, I have just changed it. Also added `ClientType: TypeAlias = Literal["vue2", "vue3"]`.

Reply:
Let's make TypeAlias for backend and exec_mode

exec_mode = [main, desktop, task, coroutine]

Reply:
I'm guessing all the typing you are doing still work in Python 3.7+ ?

Reply:
> I'm guessing all the typing you are doing still work in Python 3.7+ ?

Hmm, `TypeAlias` and `Literal` are not available with python 3.7.
- `Literal` is available since Python 3.8
- `TypeAlias` is available since Python 3.10 but not necessary

Python 3.7 is on "end of life" status and not maintained anymore: [Status of Python versions](https://devguide.python.org/versions/).
Do you still want to make it Python 3.7 compatible?

Reply:
I guess we can go to 3.8 and skip 3.7 now...

Reply:
If so, then I guess the PR is ready, I removed the `TypeAlias` typing.